app/models/consent_question.rb

```diff
@@ -52,6 +52,30 @@ class ConsentQuestion < ApplicationRecord
 
   private
 
+  def valid_answers
+    case question_type
+    when 'checkbox'
+      ['yes', 'no']
+    when 'multiple choice'
+      question_options.map { |question_option| question_option.value }
+    when 'checkbox agreement'
+      ['yes', 'no']
+    when 'multiple checkboxes'
+      question_options.map { |question_option| question_option.value }
+    end
+  end
+
+  def default_answer_is_in_question_options
+    if valid_answers.nil?
+      true
```

Comment:
Not sure I agree with the logic here... I read this as: "if the list of valid answers is empty, then that list contains the default answer".

Reply:
I think you're right... I don't remember why I wrote it like this. Reading the code again now, it should be impossible for `valid_answers` to be `nil`; The `case` statement in `valid_answers` covers every possible kind of `question_type`, and never returns `nil` in each case.

Reply:
Ah. It's possible to create (but not save) a question with an invalid `question_type`. There's a separate validator which will rightly complain about that. But `default_answer_is_in_question_options` will still be called, so it needs to handle invalid `question_type`s too. I might just rename `default_answer_is_in_question_options` to `default_answer_is_valid` or something, to make it less misleading.